system/system.proto

```diff
@@ -230,6 +231,7 @@ message TracerouteRequest {
   }
   L4Protocol l4protocol = 9;
   bool do_not_lookup_asn = 10; // Do not try to lookup ASN
+  string network_instance = 11; // Network instance of the destination
```

Comment:
```suggestion
  string network_instance = 11; // // Network instance to look up the destination in.
```

Reply:
I changed the comment a bit to be lint compliant. Please take a look.